fleetspeak/src/server/components/proto/fleetspeak_components/config.proto

```diff
@@ -75,3 +78,13 @@ message AdminConfig {
   // "localhost:1234". Required.
   string listen_address = 1;
 }
+
+message StatsConfig {
+  // Stats collector to be used. Optional, but recommended.
+  // Defaults to no stats collector.
+  string collector = 1;
```

Comment:
Yes, that was one thing I was not entirely sure about. I figured that if a user wishes to use a different type of collector, which is not Prometheus, it would be easier to later integrate it without having to add an additional flag (such as `collector`) to the config file.
But you're right, and I also think that for the open source version it suffices to have just Prometheus.

Reply:
The original design intent is that advanced users can define and compile in a collector to bridge to any monitoring system that they want to use. Non-advanced users can pick a standard implementation without recompilation, using this components proto.

I think that having a string here is strange - enum would be better - also that it is probably fine to no worry about system selection here. When we have a another case, we can decide how to adjust the config proto. e.g. if it also only requires a port, we could add an enum selector that defaults to prometheus. If it requires something else, we need to rework the proto more.